deployment/templates/fluentd.yaml

```diff
@@ -84,6 +84,9 @@ objects:
             - name: dockerhostname
               mountPath: /etc/docker-hostname
               readOnly: true
+            - name: throttle-settings
```

Comment:
what's the migration plan for existing fluentd without this secret? are we basically going to tell them to reinstall from scratch...